exchanges/btcmarkets/btcmarkets_websocket.go

```diff
@@ -358,26 +359,33 @@ func (b *BTCMarkets) Subscribe(subs subscription.List) error {
 	}
 
 	var errs error
-	for _, s := range subs {
-		if baseReq.Key == "" && common.StringSliceContains(authChannels, s.Channel) {
-			if err := b.authWsSubscibeReq(baseReq); err != nil {
-				return err
+	if authed := subs.Private(); len(authed) > 0 {
```

Comment:
While I'd appreciate a test for this func, it doesn't have the same level of changes like in Bitstamp PR and so I will not force the issue

Reply:
I think I backed away from TestSubscribe because of the low priority of this exchange.
I have it in a distant TODO, though.